labelingtool/README.md

```diff
@@ -0,0 +1,20 @@
+# dida Labeling Tool
+
+This repository contains the code to run a simple labeling tool for image classification in a Jupyter notebook.
+
+<!-- It accompanies [this article](ADD-LINK) in the dida blog. -->
```

Comment:
Link missing?

Reply:
yeah because the article is not posted yet and I didn't have the link yet

Reply:
I'll add it as soon as the article goes online